code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -224,9 +226,9 @@
 		try_say_list(say_got_target)
 		target_mob = M
 		last_target_time = world.time
-		return M
 		if(check_threat(M) >= 4)
-			global_announcer.autosay("[src] is attempting to 'detain' suspect <b>[target_name(M)]</b> in <b>[get_area(src)]</b>.", "[src]", "Security")
+			global_announcer.autosay("[src] is attempting to detain suspect <b>[target_name(M)]</b> in <b>[get_area(src)]</b>.", "SmartCollar oversight", "Security")
```

Comment:
I assume this didn't work until fairly recently, since the global_announcer couldn't speak on the security channel